MUXControls.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29009.5
+# Visual Studio 15
```

Comment:
Unintentional change?

Reply:
This came just from saving the file in VS2017, I think.  I don't think it should cause any problems.